assets/index.js

```diff
@@ -0,0 +1,22 @@
+$(function() {
```

Comment:
This can be replaced with `(function() { ... })()` as the script is evaluated after all referenced DOM nodes.
There should also be a `use strict` directive.